app/js/datepicker.jsx

```diff
@@ -445,5 +468,9 @@ function noop(data) {
     console.log("changing", data);
 }
 
+function clone(m, tz) {
+  return moment.tz(m.valueOf(), tz);
```

Comment:
Just so I know, what is the problem with `clone()` and how is it different from what we are doing here?